.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,36 @@
+name: "Issue report"
```

Comment:
@bsmth Most bugs come from people following the tutorial on MDN incorrectly then raising bugs against this repo. That's why the original issue template had the information below. Is there some way we can add it back?

----

Before posting an issue, please confirm:
- [ ] Issue can be reproduced on THIS repository (setup using [quickstart instructions](https://github.com/mdn/django-locallibrary-tutorial#quick-start)).
- [ ] I am using a clean/fresh database
- [ ] I have provided output logs and information about my environment setup below.

Reply:
>Is there some way we can add it back?

Of course, let me add it back 👍🏻

Reply:
Thanks!

Reply:
Added these checkboxes back in https://github.com/mdn/django-locallibrary-tutorial/pull/113/commits/63450e7b08b7a0f9eb3d6510535df06a79d74857